functions/bobthefish_display_colors.fish

```diff
@@ -141,5 +141,9 @@ function bobthefish_display_colors -a color_scheme -d 'Print example prompt colo
   echo -ns $desk_glyph desk ' '
   __bobthefish_finish_segments
 
+  __bobthefish_start_segment $color_aws_vault
+  echo -ns $aws_vault_glyph ' aws-vault '
```

Comment:
There's no more `$aws_vault_glyph` :)

Reply:
Looking back over my work, I realised that I did a `drop` instead of `squash` for my glyph fix-up commit. Ugh.